ideascube/serveradmin/catalog.py

```diff
@@ -35,6 +35,32 @@ def rm(path):
         shutil.rmtree(path)
 
 
+def load_from_file(path):
+    """Load catalog data from a local file
```

Comment:
I meant that only this comment worth it, the rest is noise and should be removed IMHO. :s